landing/static/js/index.js

```diff
@@ -114,25 +155,34 @@ $('#contact-form').submit(function(event){
     }
 
     $.ajax({
-        url: "https://ipa.dataocean.us/api/landing_mail/",
+        url: "https://dataocean-ipa.ml/api/landing_mail/",
```

Comment:
host should be a environment variable, for test its have one value for prod other.

Reply:
this is actual problem, you can change to https://ipa.dataocean.us temporary